2022 - Imperialis Militia.cat

```diff
@@ -10352,24 +10349,15 @@ equal to the number of Militia Gunners in the unit when making a Shooting Attack
           </conditions>
         </modifier>
       </modifiers>
-      <profiles>
-        <profile id="60d2-8915-fa6d-b478" name="Militia Land Raider Proteus Carrier" publicationId="a716-c1c4-7b26-8424" page="76" hidden="false" typeId="2fae-b053-3f78-e7b2" typeName="Vehicle">
-          <characteristics>
-            <characteristic name="Unit Type" typeId="e555-4aed-dfcc-c0b4">Vehicle (Transport, Reinforced, Third-line)</characteristic>
-            <characteristic name="Move" typeId="3614-4a2d-bffb-90e4">12</characteristic>
-            <characteristic name="BS" typeId="51fb-b7d9-aa59-863d">3</characteristic>
-            <characteristic name="Front" typeId="0ef8-a648-01d0-08ee">14</characteristic>
-            <characteristic name="Side" typeId="f150-c0dc-c192-9cb3">14</characteristic>
-            <characteristic name="Rear" typeId="8d4e-2aea-fffc-d556">14</characteristic>
-            <characteristic name="HP" typeId="a76c-83b1-602f-9e62">5</characteristic>
-            <characteristic name="Transport Capacity" typeId="0c90-79e2-f768-e547">12</characteristic>
-            <characteristic name="Access Points" typeId="e217-1b1e-9494-3e3e">One on each side of the hull and one at the front.</characteristic>
-          </characteristics>
-        </profile>
-      </profiles>
       <infoLinks>
         <infoLink id="7469-12ea-cba1-0625" name="Assault Vehicle" hidden="false" targetId="aa61-11f6-2bb5-7c0e" type="rule"/>
         <infoLink id="4125-edd1-8483-d061" name="Power of the Machine Spirit" hidden="false" targetId="5a93-13e0-809d-782a" type="rule"/>
+        <infoLink name="Unknown" hidden="false" type="profile" id="e6ef-ab28-c5aa-319e" targetId="2a46-a372-d986-d0eb">
```

Comment:
Name = "Unknown", does this name display? 
Also, for all of these, do we want to modifier set name="<faction> <whatever>"

Reply:
It doesn't display as far as I can see

In fact I can't actually see the "unknown" in the data editor.

Reply:
Looks like github tried to interpret parts of my comment as HTML and so what I meant to say with the second line was not clear. 
I was saying we should set the name on all the profiles with `set name=<faction> <unit name>`
Sidenote is there a prepend option?